src/content/pages/photos.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Photos
+subtitle: EuroPython 2024 in Pictures
+---
+
+# Official Photos
+Check out the official photos from EuroPython 2024! - Enjoy and relive the memories and highlights of the week of Python in Prague & Remote - and share them with your friends and colleagues.
+<div style={{textAlign: "center"}}>
+    <ButtonLink href="https://www.flickr.com/photos/europython/collections/72157723111210737/">Collection of the EuroPython2024 Official Conference Photos</ButtonLink>
+</div>
+
+The official conference Photos where taken by:
+- [Jan Kaderka](www.jankaderka.cz)
```

Comment:
Can we add something like:

```suggestion
The Conference and Social Event photos where taken by:
- [Jan Kaderka](www.jankaderka.cz)

The photos from Sprint days were taken by our community photographers:
- George Zisopoulos
- Moisés Guimaraẽs
```

Reply:
The URL needs to be https://www.jankaderka.cz/. Otherwise, it doesn't resolve. You can probably edit Laís' comment to do the auto-commit thing.